src/App.vue

```diff
@@ -0,0 +1,40 @@
+<template lang="pug">
+  #app
+    Calculator
+</template>
+
+<script>
+import Calculator from "./components/Calculator.vue";
+
+export default {
+  name: "App",
+  components: {
+    Calculator
+  },
+  data() {
```

Comment:
I'm a bit green to Vue so not sure, but I think this data function may be able to be removed

Reply:
artifact from having the calculator on this page